packages/email-sender/src/services/emailSenderProcessor.ts

```
@@ -1,64 +1,88 @@
 /* eslint-disable functional/no-let */
 import { genericInternalError } from "pagopa-interop-models";
 import { EachMessagePayload } from "kafkajs";
-import { delay, EmailManagerSES, Logger } from "pagopa-interop-commons";
+import { delay, EmailManagerSES, Logger, logger } from "pagopa-interop-commons";
 import Mail from "nodemailer/lib/mailer/index.js";
-import { TooManyRequestsException } from "@aws-sdk/client-sesv2";
+import {
+  LimitExceededException,
+  SendingPausedException,
+  TooManyRequestsException,
+} from "@aws-sdk/client-sesv2";
+import { EmailNotificationPayload } from "../model/emailNotificationPayload.js";
 import { config } from "../config/config.js";
 
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
 export function emailSenderProcessorBuilder(
-  loggerInstance: Logger,
-  sesEmailManager: EmailManagerSES,
-  sesSenderData: { label: string; mail: string }
+  sesSenderData: {
+    label: string;
+    mail: string;
+  },
+  sesEmailManager: EmailManagerSES
 ) {
   return {
     async processMessage({
       message,
       partition,
     }: EachMessagePayload): Promise<void> {
-      let attempts: number = 0;
+      if (!message.value) {
+        throw genericInternalError(
+          `Empty message for partition ${partition} with offset ${message.offset}`
+        );
+      }
+
+      let jsonPayload: EmailNotificationPayload;
+      let loggerInstance: Logger;
+      let mailOptions: Mail.Options;
+      try {
+        jsonPayload = JSON.parse(message.value.toString());
```

This could fail if the json is not well formatted, so I would discard the error without throwing

---

Fixed, I followed the same logic as above with the empty message